shuffle/cli/src/new.rs

```diff
@@ -15,14 +15,17 @@ pub const DEFAULT_BLOCKCHAIN: &str = "goodday";
 /// Directory of generated transaction builders for helloblockchain.
 const EXAMPLES_DIR: Dir = include_dir!("../move/examples");
 
-pub fn handle(blockchain: String, pathbuf: PathBuf) -> Result<()> {
+pub fn handle(home: &Home, blockchain: String, pathbuf: PathBuf) -> Result<()> {
     let project_path = pathbuf.as_path();
     println!("Creating shuffle project in {}", project_path.display());
     fs::create_dir_all(project_path)?;
 
     let config = shared::ProjectConfig::new(blockchain);
     write_project_files(project_path, &config)?;
     write_example_move_packages(project_path)?;
+    home.generate_shuffle_path_if_nonexistent()?;
+    // Writing default localhost network into Networks.toml
+    home.write_default_networks_config_into_toml()?;
```

Comment:
Does this always happen even if the Networks.toml file already exists? Would we overwrite Networks.toml on creation of my second project. Why does new::handle need home anyways?

Reply:
I've changed `write_default_networks_config_into_toml` to `write_default_networks_config_into_toml_if_nonexistent` which checks if the file exists before writing into it.

Reply:
Before the .shuffle folder was generated whenever a shuffle node was used. But if a user creates a new project, doesn't use shuffle node because they want to directly connect to trove testnet, and creates an account to deploy to testnet, there is no .shuffle folder created. So I think it's a good idea to generate the .shuffle folder when a new project is created. The reason I pass it into the handle is because we have a top level home created in the main method that has all the proper initializations that can be used across many files that need it. Since shuffle new now needs to check if the .shuffle folder, we can use the home that's generated in main and not create another instance of it.